test/model/src/core/kv_transaction.cpp

```diff
@@ -136,6 +136,13 @@ kv_transaction::prepare(timestamp_t prepare_timestamp)
     if (state() != kv_transaction_state::in_progress)
         throw model_exception("The transaction must be in progress");
 
+    /* Ensure that the transaction does not include updates to non-timestamped tables. */
+    for (auto &u : _updates)
```

Comment:
```suggestion
    for (const auto &u : _updates)
```

Reply:
I think there are other `for` loops that could use `const auto&` but I am aware this is not within the scope of that ticket.